### PR TITLE
Develop

### PR DESCRIPTION
**Pull request by Mmmams**
I want to merge master with develop, for the master branch has the newest features and fixed bugs.

### CODE REVIEW

File: src/components/list/list.component.jsx [hunk 1]
```diff
@@ -1,11 +1,10 @@
-import React, { createRef } from "react";
```
Comment (slolkunchik) [inline src/components/list/list.component.jsx]: prioritize libraries first, then your local files

File: src/components/list/list.component.jsx [hunk 2]
```diff
@@ -15,33 +14,47 @@ import {
   decreaseExtraScore,
   increaseScore,
   changeColor,
+  getRandom,
 } from "../../redux/birds/birds.action";
 
 import { useDispatch, useSelector } from "react-redux";
```
Comment (slolkunchik) [inline src/components/list/list.component.jsx]: should be near react

File: src/components/list/list.component.jsx [hunk 3]
```diff
@@ -15,33 +14,47 @@ import {
...
 
 import right from "../../assets/true.mp3";
 import wrong from "../../assets/wrong.mp3";
+import { useEffect } from "react";
```
Comment (slolkunchik) [inline src/components/list/list.component.jsx]: move it to line 1

File: src/components/qustion/question.component.jsx [hunk 4]
```diff
@@ -17,8 +16,12 @@ const Question = () => {
   const randomVal = useSelector(selectRandom);
   const answered = useSelector(selectAnswered);
   const nameList = useSelector(selectBirdsNames);
-  const title = useSelector(selectTitle);
   const birdsInfo = useSelector(selectBirdsInfo);
+  const currentAudio = useRef(null);
+  if (answered === true) {
```
Comment (slolkunchik) [inline src/components/qustion/question.component.jsx]: empty line above

File: src/redux/birds/birds.sagas.js [hunk 5]
```diff
@@ -166,6 +166,16 @@ export function* goNextLevel() {
         )
           .then((response) => response.json())
           .then((value) => data.push(value));
+        yield fetch(
+          `https://www.flickr.com/services/rest/?method=flickr.photos.search&api_key=7ecd0010d836ddcdfb59aa1364622ed5&tag_mode=all&extras=url_m&format=json&nojsoncallback=1&tags=свиристель`
```
Comment (slolkunchik) [inline src/redux/birds/birds.sagas.js]: don't repeat yourself: 
- use constants for the first part of the URL, 
- add function for yield fetch (the code is repetitive)
- API key uses from env file (you need dotenv library)

File: src/redux/birds/birds.sagas.js [hunk 6]
```diff
@@ -166,6 +166,16 @@ export function* goNextLevel() {
...
+        )
+          .then((response) => response.json())
+          .then((value) => data.push(value));
+        yield fetch(
+          `https://www.flickr.com/services/rest/?method=flickr.photos.search&api_key=7ecd0010d836ddcdfb59aa1364622ed5&tag_mode=all&extras=url_m&format=json&nojsoncallback=1&tags=иволга`
```
Comment (slolkunchik) [inline src/redux/birds/birds.sagas.js]: and here

**Review by slolkunchik — CHANGES_REQUESTED** [review]
your saga code is too repetitive (don't repeat yourself), apply my comments for all sagas and move libraries to the top of the module

**Review by slolkunchik — COMMENTED** [review]
1) update readme file - it is not about your project now
2) I suggest using airbnb eslint
3) add material Ui 
4) birdsAudio and birdsNames move to separate file with data
5) add an empty line before 'return'

**Comment by Mmmams** [comment]
close pull req bcs update my branch

**Pull request closed without merging**